CONTRIBUTING.md

```diff
@@ -19,8 +19,9 @@ All types of contributions are encouraged and valued. See the [Table of Contents
 - [I Want To Contribute](#i-want-to-contribute)
   - [Reporting Bugs](#reporting-bugs)
   - [Suggesting Enhancements](#suggesting-enhancements)
+  - [Project Structure](#project-structure)
   - [Your First Code Contribution](#your-first-code-contribution)
-    - [Setting Up Your Development Environment](#setting-up-your-development-environment)
+    - [Setting up your development environment](#setting-up-your-development-environment)

```

Comment:
This copy change is now non-uniform to the rest of the options here.  They should all be capitalized or all not capitalized, but not a mixture

Reply:
oops, sorry for preemptive merge, would you like to be tagged as a reviewer on all doc changes to the repo @kn0wmad

Reply:
@kn0wmad nice spot. I fixed up the inconsistent casing in #1467.